NEWS.md

```diff
@@ -1,3 +1,11 @@
+v0.24 (2016-08-22)
+------------------
+- The `fixed` key is now recognized in limits
+- new column in output of `excelexport` and `tableexport` commands to
+  indicate if reaction of compounds are in the model
+- Can input long strings, dicts, or open files as model
+- Do not return values for zero mass compounds
```

Comment:
"Zero mass compounds are now omitted from the output of the `massconsistency` command."